### ISSUE TITLE
[Question] Error when sending messages to bot on specific Skype for Business tenant

### ISSUE THREAD

**Issue by johnkuefler** [Microsoft/BotBuilder#2820]
I'm having an issue with a Skype for Business bot in one of my Office 365 tenants. I have multiple tenants and in the first one I was able to add a Skype for Business bot framework bot with no issues. My second tenant is a federated education tenant - the first tenant is a non federated developer tenant. When I add the bot to the federated tenant and attempt to send messages to it (even after 24 hours of waiting), Skype for Business shows an error under each message I try to send to it - "The action couldn’t be completed. Please try again later." 
![image](https://cloud.githubusercontent.com/assets/13266355/26411408/757f86a4-406b-11e7-95fc-26724df6cef6.png)

I created the bot with a uri of sip:username@domain.onmicrosoft.com, not using my custom federated domain name. Any ideas what might be going on? 

EDIT: Posted to StackOverflow as well: https://stackoverflow.com/questions/44148206/error-when-sending-messages-to-skype-for-business-bot-framework-bot

**Comment by nwhitmont** [comment]
@johnkuefler are you using .NET or Node SDK?

**Comment by johnkuefler** [comment]
@nwhitmont I am using .NET.

**Comment by sab435** [comment]
I am facing this **exact** same issue with a federated tenant. 

I am using the Node SDK.

**Comment by johnkuefler** [comment]
I was able to resolve this. It seems like it has something to do with when you add a bot to a federated domain utilizing the custom domain name. This errors out when you use the New-CsOnlineApplicationEndpoint -Uri bot@domain.com. However, it still creates a bad endpoint. The cmdlet does not error out if you use a uri like bot@domain.onmicrosoft.com. However, after I had already added the bad endpoint, it seemed like no endpoints would work after that. To resolve, I removed all endpoints, including the first bad one with the uri bot@domain.com. Waited 8 hours, added a single endpoint using the onmicrosoft.com Office 365 domain, waited probably about 12 hours, and it started working.
